cmd/algorand-indexer/main.go

```diff
@@ -110,6 +110,7 @@ func init() {
 
 	rootCmd.AddCommand(importCmd)
 	rootCmd.AddCommand(daemonCmd)
+	rootCmd.AddCommand(resetCmd)
```

Comment:
I'm not sure we should advertise this feature.
```suggestion
	rootCmd.AddCommand(resetCmd)
	resetCmd.Hidden = true
```